src/plugins/UtilCommands/controllers/commands/createUser.js

```diff
@@ -16,11 +16,13 @@ exports.conf = {
 
 exports.run = async (client, message, args) => {
   let userId = args[0];
-  let userInDb = await User.findByPk(userId);
+  let userInDb = await User.findOne({ where: { discordId: userId } });
```

Comment:
Should we rename all userId to discordId?

Reply:
This right here is just a test command I made to make sure the auto incrementing id would work. It will all be removed.